PROPOSAL.md

```diff
@@ -19,10 +19,11 @@ with a three out of five signature threshold, with the following custodians:
 
 The wallet shall operate as follows:
 
-1. With every release of Dogecoin Core, 500,000 DOGE shall be distributed
-   among all contributors credited in the release notes, exclusively. These
-   are the only permissible transfers out from this wallet except for a
-   dissolution under clause six.
+1. With every major or minor release of Dogecoin Core, 500,000 DOGE shall be
+   distributed among all contributors credited in the release notes,
```

Comment:
Maybe add "credited in the release notes *since the previous major or minor release*" to make our desired intent completely clear?

Reply:
Done